srcpkgs/keepassxc/template

```diff
@@ -19,7 +19,7 @@ maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
 license="GPL-3, GPL-2, BSD-3-clause, LGPL-2, LGPL-2.1, LGPL-3+, Boost-1.0, CC-0"
```

Comment:
please use SPDX identifier

Reply:
To quote from https://github.com/keepassxreboot/keepassxc/blob/develop/COPYING

> This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 2 or (at your option)
version 3 of the License.

So `GPL-2.0-or-later` would be the correct form? I presume the other licenses are listed for the other assets. Should I include those as well?

Reply:
make a choice, GPL-2.0-only or GPL-3.0-only are both correct

Reply:
I've deferred the question via IRC to the original developers of KeePassXC for their thoughts.

> I presume the other licenses listed in the srcpkg [and the project's COPYING file] are for the other assets. Should I include those as well?

Still wondering the best way to handle this.

Reply:
Had a chat with a KeePassXC developer.

```
10:24 <JTL> Anyone from the keepassxc team have any thoughts regarding this?
10:24 <JTL> https://github.com/void-linux/void-packages/pull/2076/files/a418f4071b661fa483215b2a90fa959569e479af
10:43 <droidmonkey> Gpl 3
13:53 <JTL> droidmonkey: as in just GPL 3?
13:53 <JTL> and no other licenses?
17:50 <droidmonkey> KeePassXC code is preferred GPLv3
17:51 <droidmonkey> All other code is as marked and in COPYING
17:51 <droidmonkey> GPLv2 is only included due to KeePassX
18:17 <JTL> Thanks for your clarification. I will update the PR tonight and quote you.
```

I updated the template to state `GPL-3.0-only`.